Sources/SwiftInspectorVisitors/NestableTypeVisitor.swift

```diff
@@ -88,9 +88,18 @@ public final class NestableTypeVisitor: SyntaxVisitor {
   }
 
   public override func visit(_ node: TypealiasDeclSyntax) -> SyntaxVisitorContinueKind {
-    let typealiasVisitor = TypealiasVisitor(parentType: parentType)
-    typealiasVisitor.walk(node)
+    guard
+      !topLevelParsingTracker.hasFinishedParsing,
+      let topLevelDeclarationName = topLevelDeclaration?.nestableInfo.name
+    else {
+      assertionFailure("Encountered more than one top-level declaration. This is a usage error: a single NestableTypeVisitor instance should start walking only over a declaration syntax node")
+      return .skipChildren
+    }
 
+    // We've previously found an top-level declaration, so this must be an inner declaration.
```

Comment:
```suggestion
    // We've previously found a top-level declaration, so this must be an inner declaration.
```

Reply:
this was copy-paste. fixed in both places 77bdc92e6b7f0f2647420f09e5f33c0ec1c10ddd